examples/01_simple_agent_example/README.md

```
@@ -11,11 +11,12 @@ This example shows you how to build a basic video AI agent using [Vision Agents]
 
 - Python 3.13 or higher
 - API keys for:
-  - [OpenAI](https://openai.com) (for the LLM)
-  - [Cartesia](https://cartesia.ai/) (for text-to-speech)
+  - [Gemini](https://aistudio.google.com/) (for the LLM)
+  - [ElevenLabs](https://elevenlabs.io/) (for text-to-speech)
   - [Deepgram](https://deepgram.com/) (for speech-to-text)
   - [Stream](https://getstream.io/) (for video/audio infrastructure)
-  - [Smart Turn](https://fal.ai/models/fal-ai/smart-turn) (for turn detection)
+  - [Hugging Face]()https://huggingface.co/ to access [Vogent 
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)